libvast/include/vast/pipeline.hpp

```diff
@@ -179,6 +211,31 @@ class pipeline final : public operator_base {
   auto predicate_pushdown(expression const& expr) const
     -> std::optional<std::pair<expression, operator_ptr>> override;
 
+  template <class Inspector>
+  friend auto inspect(Inspector& f, pipeline& x) -> bool {
+    if constexpr (Inspector::is_loading) {
+      auto repr = std::string{};
+      if (not f.object(x)
+                .pretty_name("vast.logical-pipeline")
```

Comment:
The pretty name here and below should just be `vast.pipeline`, I think, as the class is no longer named `logical_pipeline` either.